recipe/build.sh

```diff
@@ -8,7 +8,6 @@ cmake ${CMAKE_ARGS} \
     -GNinja \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_PREFIX=$PREFIX \
-    -DMSGPACK_CXX11=YES \
```

Comment:
looking at their cmake file, this seems to disable the static builds, helping with the test to assert lack of static binaries.